apps/condo/domains/user/utils/directAccess/server.utils.js

```diff
@@ -90,7 +111,26 @@ async function canDirectlyReadSchemaObjects (user, schemaName) {
  * @return {Promise<boolean>}
  */
 async function canDirectlyManageSchemaObjects (user, schemaName) {
-    return await _hasSpecificRight(user, generateManageSchemaFieldName(schemaName))
+    return await _hasSpecificRights(user, [generateManageSchemaFieldName(schemaName)])
+}
+
+const FIELD_NAMES_TO_SKIP_ACCESS = ['dv', 'sender']
+
+async function canDirectlyManageSchemaFields (user, schemaName, originalInput, operation) {
+    const isUpdateOperation = operation === 'update'
+    const fieldNamesWithAccess = getFieldNamesBySchemaName(schemaName)
+
+    if (isUpdateOperation && isArray(fieldNamesWithAccess) && !isEmpty(fieldNamesWithAccess)) {
```

Comment:
Why `isUpdate...` filter here?

Reply:
You can only update a field, you cannot create or delete it (as a schema object). 

Even if there is only one field in schema, then you have access to control only this one field, and not entire schema object

Reply:
Seems like I don't get it:
1. If fields-access function resolves to `false` in `create` operation, then it means that I cannot pass this fields in `create` mutation.
2. So, what if I have schema `A`, and `canManageA: true` and `canManageAField: true`? Seems like I should be able to create entities of `A` with `Field`, shouldn't I?

Reply:
You should check accesses in the following sequence:
1) `canDirectlyManageSchemaObjects` - Direct access to object management (creating/deleting an object and updating its fields)
2) `canDirectlyManageSchemaFields` - Direct access to updating object fields
3) Default accesses

It looks like it's really worth combining these functions together so that the order doesn't change

- If you have `canManageA: true`, then it doesn’t matter what value you have specified for `canManageAField`. You can create/update/delete objects.

- If you have `canManageA: false` and `canManageAField: true`, then you can only update the specified fields.

- If you have `canManageA: false` and `canManageAField: false`, then you will not have direct access to any actions that bypass standard checks